pkgs/development/libraries/opencolorio/default.nix

```diff
@@ -1,22 +1,33 @@
-{ stdenv, lib, fetchurl, cmake, unzip, boost }:
+{ stdenv, lib, fetchurl, cmake, unzip, boost, pkgconfig, lcms2, tinyxml, git }:
+
+with lib;
 
 stdenv.mkDerivation rec {
   name = "opencolorio-${version}";
   version = "1.1.0";
 
   src = fetchurl {
```

Comment:
Try using `fetchFromGitHub` it is preferred over `fetchurl` and will probably allow you to get rid of `unzip`.

Reply:
Done. Thanks.